src/main/java/com/emc/pravega/perf/PerfStats.java

```diff
@@ -53,59 +50,48 @@ public PerfStats(String action, long numRecords, int reportingInterval, int mess
             this.windowStart = 0;
             this.index = 0;
             this.iteration = 0;
-            this.sampling = (int) (numRecords / Math.min(numRecords, 500000));
-            this.latencies = new long[(int) (numRecords / this.sampling)];
-            this.index = 0;
+            this.latencies = new double[(int) (numRecords / reportingInterval)];
```

Comment:
Can you explain the change? IMO we will lose track of the earlier latencies and use only final reporting window to report overall latency. This may not show a true picture.

Reply:
hi Arvind
   Thanks for your review comments..
This is written with intention that my sampling time is same as reporting time... 
for example, if i have set the runtime as 1 minute and reporting interval is 1 sec, for every 1 sec i will collect the latency sample..   so , in 1 minute , I will collect the 60 samples if the reporting interval is set to 1 sec...